Automattic/vip
# Remove enums from GraphQL codegen and make sure that prettier actually runs

## Description

Looks like prettier didn't run during the last PR because the generated files was ignored by prettier.

So I'm re-enabling it here.

We also remove enums from this PR because enums are actual objects in TypeScript - not just a type. So someone importing the enum i.e. `SomeEnum.SOME_VALUE` would get an undefined error, since we put it in a `.d.ts` instead of a `.ts` as we definitely don't want to increase our bundle size!

Hopefully this is the last of it. The formatting looks good now (spaces everywhere).

We missed this because the tests rightfully also ignore these files.

We can review this by ignoring all `.d.ts` files generated - but pick one to verify the formatting is good.

## Steps to Test

Similar to the previous PR.

1. Check out PR.
1. Get a hold of `schema.gql` and paste it in project root - this is the schema of the endpoint that
   we communicate with.
1. Run `npm run typescript:codegen:install-dependencies` - this will install the codegen
   dependencies without updating `package.json`
1. Run `npm run typescript:codegen:generate` - this will regenerate the types.
1. There shouldn't be any new files to commit (which means prettier ran through graphql-codegen)
1. Verify that the generated files conform to the style generated in non-generated files.
